app/feature/feature-choose-tier/src/main/kotlin/com/hedvig/android/feature/change/tier/ui/stepsummary/SummaryDestination.kt

```diff
@@ -1,18 +1,345 @@
+        enabled = true,
+        buttonStyle = Secondary,
+        buttonSize = ButtonSize.Medium,
+      )
+    }
+  }
+}
+
+@Composable
+fun DisplayItemRowSecondaryColor(leftText: String, rightText: String) {
+  HorizontalItemsWithMaximumSpaceTaken(
+    startSlot = {
+      HedvigText(leftText, color = HedvigTheme.colorScheme.textSecondary)
+    },
+    endSlot = {
+      Row(
+        horizontalArrangement = Arrangement.End,
+        verticalAlignment = Alignment.CenterVertically,
+      ) {
+        HedvigText(
+          rightText,
+          color = HedvigTheme.colorScheme.textSecondary,
+          textAlign = TextAlign.End,
+        )
+      }
+    },
+    spaceBetween = 8.dp,
+  )
+}
+
+@Composable
+private fun ExtendedCardContent(
+  isPDFLoading: Boolean,
+  quote: TierDeductibleQuote,
+  downloadFromUrl: (url: String) -> Unit,
+) {
+  Column {
+    Spacer(Modifier.height(16.dp))
+    HorizontalDivider()
+    Spacer(Modifier.height(16.dp))
+    HedvigText(stringResource(R.string.TIER_FLOW_SUMMARY_OVERVIEW_SUBTITLE))
+    quote.displayItems.forEachIndexed { _, item ->
+      DisplayItemRowSecondaryColor(item.displayTitle, item.displayValue)
+    }
+    Spacer(Modifier.height(16.dp))
+    HedvigText(stringResource(R.string.TIER_FLOW_SUMMARY_COVERAGE_SUBTITLE))
+    quote.productVariant.insurableLimits.forEachIndexed { i, insurableLimit ->
+      DisplayItemRowSecondaryColor(insurableLimit.label, insurableLimit.limit)
+    }
+    Spacer(Modifier.height(16.dp))
+    Row(verticalAlignment = Alignment.CenterVertically) {
+      HedvigText(stringResource(R.string.TIER_FLOW_SUMMARY_DOCUMENTS_SUBTITLE))
+      Spacer(Modifier.width(16.dp))
+      AnimatedVisibility(isPDFLoading) { HedvigThreeDotsProgressIndicator() }
+    }
+    quote.productVariant.documents.forEach { document ->
+      DocumentRow(
+        name = document.displayName,
+        downloadFromUrl = {
+          downloadFromUrl(document.url)
+        },
+      )
+    }
+  }
+}
+
+@Composable
+private fun DocumentRow(name: String, downloadFromUrl: () -> Unit) {
+  Row(
+    verticalAlignment = Alignment.CenterVertically,
+  ) {
+    HorizontalItemsWithMaximumSpaceTaken(
+      startSlot = {
+        Row(verticalAlignment = Alignment.CenterVertically) {
+          HedvigText(
+            color = HedvigTheme.colorScheme.textSecondary,
+            text = buildAnnotatedString {
+              append(name)
+              withStyle(
+                SpanStyle(
+                  baselineShift = BaselineShift(0.3f),
+                  fontSize = HedvigTheme.typography.label.fontSize,
+                  color = HedvigTheme.colorScheme.textSecondary,
+                ),
+              ) {
+                append("PDF")
+              }
+            },
+          )
+        }
+      },
+      endSlot = {
+        IconButton(
+          modifier = Modifier.size(24.dp),
+          onClick = {
+            downloadFromUrl()
+          },
+        ) {
```

Comment:
It may be a good idea to make the entire row here clickable, along with the icon on the right. Did it feel like the entire thing should be clickable perhaps?